index.bs

```diff
@@ -1274,11 +1274,15 @@ associated.
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialEntity">
     :   <dfn>name</dfn>
     ::  A human-friendly identifier for the entity. For example, this could be a company name for a [=[RP]=], or a
-        user's name. This identifier is intended for display.
+        user's name. This identifier is intended for display. [=Authenticators=] MUST support a 64 byte minimum length
+        for a [=PublicKeyCredentialEntity/name=] members's value. Authenticators MAY truncate a
+        [=PublicKeyCredentialEntity/name=] member's value to a length equal to or greater than 64 bytes.
 
```

Comment:
"support"  is open to interpretation. Are you requiring for example that the authenticator must be able to display 64 bytes worth of characters?

Reply:
Displaying is not applicable for UI less authenticators on authenticator side but needed for multiple accounts scenario where Platform shows the UI. So Authenticators MUST "Store". I think we don't need to be explain every type of authenticators now or come in future? I believe "support" is better word. What do you suggest?

Reply:
If you mean accept and store, then write accept and store? It is what you need while "support"  covers every possible use of the string, including displaying.

Reply:
Alright..  I changed it to "accept and store".